work/covidvu/predict.py

```diff
@@ -261,22 +276,19 @@ def _dumpPredictionCollectionAsJSON(predictionsPercentilesTS,
 def _main(countryTrainIndex,
           predictionsPercentiles = PREDICTIONS_PERCENTILES,
           ):
-    _, predictionsMeanTS, predictionsPercentilesTS, countryName = predictLogisticGrowth(
+    countryTS, predictionsMeanTS, predictionsPercentilesTS, countryName, trace, countryTSClean = predictLogisticGrowth(
```

Comment:
Let's try returning a single object that packs all these in the future, instead of a grab bag of variables.  It's hard to remember tha tthis thing has 6 potential return values, but if it returned a dictionary then we can solve this by doing somethin like:

```
prediction = predictLogisticsGrowth(trainIndex, percentiles)

someFunction(prediction['countryTS'])

# or

someFunction(prediction.countryTS)
```

Clean Coder™ stuff.